src/uwtools/drivers/forecast.py

```diff
@@ -186,27 +188,18 @@ def run(self) -> None:
             # This will not run the job.
             logging.info("Batch Script:")
             logging.info(batch_script)
-            return
+            return True
 
         # Run the job.
         if self._batch_script is not None:
             outpath = Path(run_directory) / self._batch_script
             with open(outpath, "w+", encoding="utf-8") as file_:
                 print(batch_script, file=file_)
                 batch_command = JobScheduler.get_scheduler(platform_resources).submit_command
-            subprocess.run(
-                f"{batch_command} {outpath}",
-                stderr=subprocess.STDOUT,
-                check=False,
-                shell=True,
-            )
+            result = run(cmd=f"{batch_command} {outpath}")
         else:
-            subprocess.run(
-                f"{run_command}",
-                stderr=subprocess.STDOUT,
-                check=False,
-                shell=True,
-            )
+            result = run(cmd=run_command)
+        return result.success
```

Comment:
Presumably the caller can decide what to do if the command failed, so just return a `bool` status value rather than raising an exception.